### DISCUSSION TITLE
🚀 Feature Request: Support --no-bundle in wrangler.toml for pages

### DISCUSSION THREAD

**Discussion by jamesthomp** [cloudflare/workers-sdk#11052] [Feature Requests]
### Describe the solution

It is currently possible to run: `wrangler pages deploy build --no-bundle`.

However when using the wrangler.toml file, if you add `no_bundle = true`, then the cloudflare build says:
```
02:15:15.837 | ✘ [ERROR] Running configuration file validation for Pages:
-- | --
02:15:15.837 |  
02:15:15.837 | - Configuration file for Pages projects does not support "no_bundle"
```

Can support for the no_bundle property be added to wrangler.toml so it has feature parity with the CLI?

**Comment by thescientist13** [comment]
This would be really helpful, seems like it could potentially be an escape hatch for an issue I am experiencing in https://github.com/cloudflare/workers-sdk/issues/7265#issuecomment-2480871605

**Comment by ryuheechul** [comment]
Dynamic `import()` function seems to fail due to bundling and as I'm currently relying on automatic deployment (via github hook) which leaves the only way to configure build behavior is via `wrangler.toml` (not `--no-bundle` CLI flag).

Now currently there seems to be two options for me:
a. stop using automatic deploy via Cloudflare and use another CI (e.g. Github Actions)
b. wait until this issue is resolved

Realistically, I probably have to go with the option a. but I hope this gets resolved one day though.
